content/blog/real-time-smart-city-traffic-monitoring-using-microservices-based-streaming-architecture-part-2.md

```diff
@@ -0,0 +1,331 @@
+---
+title: Real-time Smart City Traffic Monitoring Using Microservices-based
+  Streaming Architecture (Part 2)
+date: 2021-10-12T02:21:16.357Z
+author: Mathieu Dumoulin
+authorimage: /img/Avatar1.svg
+---
+```json
+"authorDisplayName": "Mathieu Dumoulin",
+"category": "machine-learning",
+"publish": "2017-01-10T06:00:00.000Z"
+```
+
+**Editor’s Note:** MapR products and solutions sold prior to the acquisition of such assets by Hewlett Packard Enterprise Company in 2019 may have older product names and model numbers that differ from current solutions. For information about current offerings, which are now part of HPE Ezmeral Data Fabric, please visit <https://www.hpe.com/us/en/software/ezmeral-data-fabric.html>
+
+***Modern Open Source Complex Event Processing For IoT***
+
+This series of blog posts details my findings as I bring to production a fully modern take on Complex Event Processing, or CEP for short. In many applications, ranging from financials to retail and IoT applications, there is tremendous value in automating tasks that require to take action in real time. Putting aside the IT system and frameworks that would support this capability, this is clearly a useful capability.
```

Comment:
add the word "one" prior to "to take action in real time.", so it reads "...automating tasks that require one to take action in real time."

Reply:
done